priv/app/function_browser.jsx

```diff
@@ -107,7 +105,7 @@ export default class FunctionBrowser extends React.Component {
   }
 
   matchFunSignature(input) {
-    var regex = /(\w+):(\w+)\/(\d+)/;
+    var regex = /([\w\.]+):(\w+)\/(\d+)/;
```

Comment:
if we want to be more strict we can do sth like this: `\w+(\.\w+)*` <-  hope this is a correct regex format :)